filters/openpolicyagent/openpolicyagent.go

```diff
@@ -404,7 +418,7 @@ func (registry *OpenPolicyAgentRegistry) NewOpenPolicyAgentInstance(bundleName s
 		return instance, nil
 	}
 
-	instance, err := registry.newOpenPolicyAgentInstance(bundleName, config, filterName)
+	instance, err := registry.newOpenPolicyAgentInstance(bundleName, *registry.config, filterName)
```

Comment:
the interpolation function could be put on the instance config now for more encapsulation, it would also allow to remove one parameter from the `new` method as this takes both the interpolated config and the config itself